include/libcyphal/transport/can/transport.hpp

```diff
@@ -87,11 +87,65 @@ class TransportImpl final : public ICanTransport
 
     void run(const TimePoint) override {}
 
+private:
+    // Until "canardMemFree must provide size" issue #216 is resolved,
+    // we need to store the size of the memory allocated.
+    // TODO: Remove this workaround when the issue is resolved.
+    // see https://github.com/OpenCyphal/libcanard/issues/216
+    //
+    struct CanardMemory
+    {
+        std::size_t size;
+        alignas(std::max_align_t) cetl::byte data[1];
+    };
+
+    cetl::pmr::memory_resource& memory_;
+    CanardInstance              canard_instance_;
```

Comment:
Shouldn't the fields go to the end of the class scope?

Reply:
they are in private section, so I though it is fine place. I'm I right that you would like almost always see PRIVATE stuff in the following order:
1. private types, aliases
2. private member functions
3. private member consts and vars
?
maybe it worth to document this at https://kb.zubax.com/pages/viewpage.action?pageId=2195699 ?